pages/project-version/pdf/views/nts/components/species-table.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { uniq, concat } from 'lodash';
+
+const groupSpeciesDetails = version => {
+  return (version.protocols || []).reduce((species, protocol) => {
+    if (protocol && protocol.speciesDetails && protocol.speciesDetails.length > 0) {
+      protocol.speciesDetails.map(details => {
+        const existingSpecies = species.find(s => s.value === details.value);
```

Comment:
Using an object keyed by the value would be a simpler and more efficient way of collecting these.

It's probably doesn't matter on the scales we're talking about though.